packages/comments/src/main.tsx

```diff
@@ -20,19 +22,16 @@ const config = {
   placeholder: import.meta.env.VITE_PLACEHOLDER,
   formIntro: import.meta.env.VITE_FORM_INTRO,
   commentsIsOpen: import.meta.env.VITE_COMMENTS_IS_OPEN != 'false',
-  commentsIsClosedText: import.meta.env.VITE_COMMENTS_IS_CLOSED_TEXT,
+  commentIsClosedText: import.meta.env.VITE_COMMENTS_IS_CLOSED_TEXT,
```

Comment:
commentsIsClosedText klinkt beter, omdat het gaat over het hele comments blok, denk ik?

Reply:
En als ik dan toch bezig ben: commentsIsOpen en dan commentsIsClosedText is imo een verwarrende naamgeving. En dan is hieronder nog isClosed ook wat ongelukkig.

Reply:
Maar goed, fröbelwerk natuurlijk